src/Tests/RegistrationTest.php

```diff
@@ -95,11 +330,42 @@ public function testAcquisitionMultiple() {
    * existing users and we are acquiring the first.
    */
   public function testAcquisitionMultipleFirst() {
-    // @todo: Change the site config to acquire the first.
+    // Set up test environment configuration,
+    $this->allowUserRegistration();
+    // Change the site config to acquire the first.
+    $this->acquisition_config->set('acquisitions.behavior_first', 1)->save();
+
+    // Test registering a new user when all existing users are decoupled.
+    // Expected result: User should acquire the first existing user.
+    $user_1 = $this->createDecoupledUser();
+    $user_2 = $this->createDecoupledUser($user_1->email_prefix);
+    $this->createDecoupledUser($user_1->email_prefix);
+    $email = $user_1->email_prefix . '@example.com';
+
+    $edit = $this->registerNewUser('', $user_1->email_prefix);
+    // @TODO There is no confirmation message when new decoupled user is created.
+
+    // Load created user and check properties.
+    $accounts = $this->getUsersByProperty(['name' => $edit['name']]);
+
+    if (empty($accounts)) {
+      $this->fail('No User accounts loaded.');
+    }
+    else {
+      $account = reset($accounts);
+      $this->assertTrue($account->isActive());
+      $this->assertEqual($email, $account->getEmail());
+      $this->assertEqual($user_1->id(), $account->id());
```

Comment:
Although I expect this would be fine in almost all cases, there is no sorting applied to the acquisition query when taking the first. Therefore this isn't really defined... I think if we check the email matches, it doesn't really matter about which ID it matches.